### PR TITLE
add uri-handler for completions

### PR DESCRIPTION
**Pull request by aviatesk**
xref: https://github.com/JunoLab/Atom.jl/pull/164

### CODE REVIEW

File: lib/runtime/completions.js [hunk 1]
```diff
@@ -77,6 +77,9 @@ class AutoCompleteProvider {
     completion.replacementPrefix = completion._prefix ? completion._prefix : prefix
     const icon = this.makeIcon(completion.icon)
     if (icon) completion.iconHTML = icon
+    // the workaround for https://github.com/atom/autocomplete-plus/issues/868, but can appear verbose
+    if (!completion.description && completion.descriptionMoreURL)
+      completion.description = 'Click the link below to show docs:'
```
Comment (aviatesk) [inline lib/runtime/completions.js]: One thing annoying around adding this feature is this:
without `description`, `descriptionMoreURL` won't work, and the workaround can appear verbose IMO.

Comment (aviatesk) [reply lib/runtime/completions.js]: this means:
image

If you feel annoying this, I can close this PR.
Can I hear your opinion, @pfitzseb ?

Comment (pfitzseb) [reply lib/runtime/completions.js]: Would
```
       completion.description = ' '
```
look too weird?

Comment (aviatesk) [reply lib/runtime/completions.js]: Yeah, autocomplete-plus package inserts an newline after `description` anyways...

Comment (pfitzseb) [reply lib/runtime/completions.js]: Then let's just fix that with 
```
diff --git a/styles/ink.less b/styles/ink.less
index fa65ab5..e5ab2f8 100644
--- a/styles/ink.less
+++ b/styles/ink.less
@@ -8,8 +8,13 @@
 @import "syntax-variables";
 
 autocomplete-suggestion-list.select-list.popover-list {
-  .suggestion-description-content {
-    overflow-y: hidden;
+  .suggestion-description {
+    line-height: 1;
+    .suggestion-description-content {
+      overflow-y: hidden;
+      display: inline;
+      white-space: normal;
+    }
   }
 }
```
image
image

**Comment by aviatesk** [comment]
Sorry, I missed to push to master ...

File: lib/runtime/completions.js (hunk 1 above)
Comment (aviatesk) [reply lib/runtime/completions.js]: Looks really good. Thanks for the fix :)

**Pull request merged**